.github/workflows/ci.yml

```diff
@@ -0,0 +1,114 @@
+# Used as inspiration: https://github.com/mvdan/github-actions-golang
+
+name: Cross-Platform
+
+on:
+  push:
+    branches: [ v2 ]
+  pull_request:
+    branches: [ v2 ]
+
+jobs:
+  test:
+    strategy:
+      # Default is true, cancels jobs for other platforms if one fails
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest, macos-latest, windows-latest ]
+        go-version: [ 1.14.x ]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v1
+      with:
+        go-version: ${{ matrix.go-version }}
+
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Install test and coverage analysis tools
+      run: |
+        go get github.com/axw/gocov/gocov
+        go get github.com/AlekSi/gocov-xml
+        go get -u github.com/jstemmer/go-junit-report
+        echo "::add-path::$(go env GOPATH)/bin"
+
+    - name: Print Go version and environment
+      run: |
+        printf "Using go at: $(which go)\n"
+        printf "Go version: $(go version)\n"
+        printf "\n\nGo environment:\n\n"
+        go env
+        printf "\n\nSystem environment:\n\n"
+        env
+
+    - name: Get dependencies
+      run: |
+        go get -v -t -d ./...
+        # mkdir test-results
+
+    # Force bash shell, this command fails on windows if using pwsh
+    - name: Build Caddy
+      working-directory: ./cmd/caddy
+      shell: bash
+      run: |
+        CGO_ENABLED=0 go build -trimpath -a -ldflags="-w -s" -v
```

Comment:
What if we set `CGO_ENABLED=0` as `env:`?

Reply:
Ah, yeah that works. That would let us avoid `shell: bash`, right?

Reply:
Yep, I believe so.